app/models/event_stream.rb

```diff
@@ -22,4 +22,6 @@ class EventStream < ApplicationRecord
   belongs_to :container_replicator
   belongs_to :container_group
   belongs_to :container_node
+
+  belongs_to :middleware_server, :foreign_key => :middleware_server_id
```

Comment:
typically the fk mention is needed when the fk field/column name is not the rails standard so user can specify the name (not in `modelname_id` format)